strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/config/Maven2LayoutProviderTestConfig.java

```diff
@@ -22,6 +22,7 @@
  * @author Przemyslaw Fusik
  */
 @Configuration
+@PropertySource(name = "mavenTestProperties", value = "classpath:maven-test.properties")
```

Comment:
What is this about? I'm not sure I follow...

Reply:
there is `maven-test.properties` file with test environment properties for Maven related tests. It used to enable maven indexer:
```
maven.indexer.enabled=true
```